ale_linters/rust/analyzer.vim

```diff
@@ -3,6 +3,7 @@
 
 call ale#Set('rust_analyzer_executable', 'rust-analyzer')
 call ale#Set('rust_analyzer_config', {})
+call ale#Set('rust_analyzer_use_local_config', 0)
```

Comment:
Prefer if this is called  `rust_analyzer_config_file` and accepts a string with the name of the desired config file with default to "rust-analyzer.json".   

From what I understand the rust-analyzer does not define any official configuration file so I expect diferent project to use different names.

Reply:
I think this is overcomplicating things and may have performance impact. Searching files recursively can be heavy depending on how deep the folder structure is.

I would recommend leaving this to single file. Users can employ other methods or plugins to create configurations specifc for each project they work on (see discussion https://github.com/dense-analysis/ale/issues/832).